cms/envs/aws.py

```diff
@@ -177,6 +177,7 @@
 
 # Theme overrides
 THEME_NAME = ENV_TOKENS.get('THEME_NAME', None)
+COMP_THEME_DIR = path(ENV_TOKENS.get('COMP_THEME_DIR', COMP_THEME_DIR))
```

Comment:
@nedbat and I discussed this, and "comprehensive theming" was the most descriptive name that we could come up with for this feature. (It can't just be the "theming" feature, because we already have several different theming features in the codebase. :sob:) Ned said that "comprehensive theming" was too long to type all the time, and suggested that we shorten it to "comp theming", which is where this variable name came from.

Do you have a suggestion for a better, less confusing name? I'm certainly open to suggestions.